.appveyor.yml

```diff
@@ -0,0 +1,48 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits unless the first line of the
+# commit message contains '[ci skip]'
+# Enable 'Do not build on "Push" events' in the AppVeyor project settings
+# to only build commits from pull requests
+branches:
+  only:
+    - master
+
+only_commits:
+  files:
+    - .appveyor.yml
+    - content/
+
+image: ubuntu
+services:
+  - docker
+
+install:
+  - source ci/install.sh
+  - env
+    
+test_script:
+  - bash build/build.sh
+  - MANUSCRIPT_FILENAME=manuscript-$APPVEYOR_BUILD_VERSION-${APPVEYOR_REPO_COMMIT:0:7}.pdf
```

Comment:
Because pull request builds make a merge commit rather than just build the commit from the PR, the commit shown in MANUSCRIPT_FILENAME is currently the merged commit rather than the commit that shows up in the PR. Perhaps it would be better for user recognition to use something like:

```bash
TRIGGERING_COMMIT = ${APPVEYOR_PULL_REQUEST_HEAD_COMMIT:-APPVEYOR_REPO_COMMIT}
MANUSCRIPT_FILENAME=manuscript-$APPVEYOR_BUILD_VERSION-${TRIGGERING_COMMIT:0:7}.pdf
```

Reply:
Done in bcd55532168cbe77ab4e0c9453539ae8b6f88aea

I like this better as long as we also use the triggering commit in the notification message below.